images/windows/scripts/tests/WindowsFeatures.Tests.ps1

```diff
@@ -66,10 +66,10 @@ Describe "Windows Updates" {
         "$env:windir\WindowsUpdateDone.txt" | Should -Exist
     }
 
-    $testCases = Get-WindowsUpdateStates | Sort-Object Title | ForEach-Object {
```

Comment:
oh I see, yeah this jSON is going to be really long I got you one second.
[internal.windows-2025.json](https://github.com/user-attachments/files/18279003/internal.windows-2025.json)

Reply:
to start we going to take this

{
  "NodeType": "HeaderNode",
  "Title": "Windows Server 2025",
  "Children": [
    {
      "NodeType": "ToolVersionNode",
      "ToolName": "OS Version:",
      "Version": "10.0.26100 Build 2605"
    },
    {
      "NodeType": "ToolVersionNode",
      "ToolName": "Image Version:",
      "Version": "20241215.1.0"
    },
    {
      "NodeType": "HeaderNode",
      "Title": "Windows updates",
      "Children": [
        {
          "NodeType": "ToolVersionNode",
          "ToolName": "Update Title",
          "Version": "Some Update",
          "TimeCreated": "2024-12-30T12:34:56Z"
        },
        {
          "NodeType": "ToolVersionNode",
          "ToolName": "Another Update",
          "Version": "Another Update Version",
          "TimeCreated": "2024-12-30T12:34:56Z"
        }
      ]
    }
  ]
}

and combine it into the prototype.

Reply:
{
  "NodeType": "HeaderNode",
  "Title": "Windows Server 2025",
  "Children": [
    {
      "NodeType": "ToolVersionNode",
      "ToolName": "OS Version:",
      "Version": "10.0.26100 Build 2605"
    },
    {
      "NodeType": "ToolVersionNode",
      "ToolName": "Image Version:",
      "Version": "20241215.1.0"
    },
    {
      "NodeType": "HeaderNode",
      "Title": "Windows updates",
      "Children": [
        {
          "NodeType": "ToolVersionNode",
          "ToolName": "Update Title",
          "Version": "Some Update",
          "TimeCreated": "2024-12-30T12:34:56Z"
        },
        {
          "NodeType": "ToolVersionNode",
          "ToolName": "Another Update",
          "Version": "Another Update Version",
          "TimeCreated": "2024-12-30T12:34:56Z"
        }
      ]
    }
  ]
}

thank you CoPIlot for intergrating this by the way!
https://github.com/bearycool11/pmll_blockchain/blob/main/.devcontainer/internal.windows-2025.json

now turns into this repo's version of this.